skunkworks_crow/src/main/java/org/odk/share/fragments/SentInstancesFragment.java

```diff
@@ -162,7 +162,7 @@ private void setupAdapter() {
 
     private void setEmptyViewVisibility(String text) {
         if (transferInstanceList.size() > 0) {
-            buttonLayout.setVisibility(View.VISIBLE);
+            buttonLayout.setVisibility(View.INVISIBLE);
```

Comment:
If button layout is not going to be visible in this fragment, then why not do it just once in onCreateView?

Reply:
there will be 2-3 lines of code to be removed if I do it from onCreate

Reply:
I agree that your choice of editing least number of lines was reasonable. But that creates a redundancy which must be resolved

Reply:
yes sure